src/lib/EventDispatcher/EventListener/ContentSuggestionSubscriber.php

```diff
@@ -73,4 +77,43 @@ public function onBuildSuggestionCollectionEvent(
 
         return $event;
     }
+
+    /**
+     * @return \Ibexa\Contracts\Core\Repository\Values\Content\Query\SortClause[]
+     */
+    private function getSortClauses(): array
```

Comment:
Is there no way to use same pattern as in `\Ibexa\Search\SortingDefinition\Provider\NameSortingDefinitionProvider` for it?

Reply:
Corrected. Now the class is utilizing `SortingDefinitionRegistry`.